client/src/components/application/NonlinearStepper/GeneralInfo.js

```diff
@@ -76,20 +75,24 @@ const GeneralInfos = ({ handleFormInput, formValues, errorMsg }) => {
         rowsMin={4}
         value={formValues.motivation}
       />
+      <br />
+
+      <Typography className="label">Mobile Number</Typography>
```

Comment:
Another approach you could try (maybe next time) instead of having `<br/>` tags above every label is to give `.label` some `margin-top` in css.